source/features/useful-not-found-page.tsx

```diff
@@ -36,54 +38,57 @@ function parseCurrentURL(): string[] {
 	return parts;
 }
 
-// If the resource was deleted, link to the commit history
-async function addCommitHistoryLink(bar: Element): Promise<void> {
-	const parts = parseCurrentURL();
-	parts[2] = 'commits';
-	const url = '/' + parts.join('/');
-	if (await is404(location.origin + url)) {
+async function getLatestChangeToFile(): Promise<Record<string, any> | void> {
+	const {branch, filePath} = new GitHubURL(location.href);
+	if (!branch || !filePath) {
 		return;
 	}
 
-	bar.after(
-		<p className="container mt-4 text-center">
-			See also the {getType()}’s <a href={url}>commit history</a>.
-		</p>,
-	);
-}
-
-// If the resource exists in the default branch, link to it
-async function addDefaultBranchLink(bar: Element): Promise<void> {
-	const parts = getCleanPathname().split('/');
-	const branch = parts[3];
-	if (!branch) {
+	const commitsResponseObject = await api.v4(`
```

Comment:
```suggestion
	const {repository: commitsResponseObject} = await api.v4(`
```

@fregante any reason were not doing this?

Reply:
No reason. It can just use the `repository` name though, otherwise we save no characters

Reply:
Done; changed it to `const {repository} = ...`.